src/main.cpp

```diff
@@ -113,7 +135,7 @@ int main(int argc, char** argv) {
     Debug::log(LOG, "Hyprland init finished.");
 
     // If all's good to go, start.
-    g_pCompositor->startCompositor();
+    g_pCompositor->startCompositor(socketName, socketFd);
```

Comment:
since we require both to work, would be nice to print an error if only one is set

Reply:
Thanks, fixed!